meteor/client/ui/RundownView/RundownTiming/RundownTimingProvider.tsx

```diff
@@ -412,7 +419,22 @@ export const RundownTimingProvider = withTracker<
 					this.partDisplayDurations[partInstancePartId] = partDisplayDuration
 					this.partDisplayDurationsNoPlayback[partInstancePartId] = partDisplayDurationNoPlayback
 					startsAtAccumulator += this.partDurations[partInstancePartId]
-					displayStartsAtAccumulator += this.partDisplayDurations[partInstancePartId] // || this.props.defaultDuration || 3000
+
+					if (playlist.previousPartInstanceId !== partInstance._id) {
+						displayStartsAtAccumulator += this.partDisplayDurations[partInstancePartId]
+					} else {
+						if (this.prevPartId !== unprotectString(playlist.previousPartInstanceId)) {
```

Comment:
See the comment on line 124. There isn't any reason to unprotect these Ids and lose type safety.

Reply:
If not i get this error: (i actually just copied this from line 414 where the same way is done on partInstance.part._id:
```
414    const partInstancePartId = unprotectString(partInstance.part._id)
```

![image](https://user-images.githubusercontent.com/29782499/121312429-a5495c80-c905-11eb-9e89-e4d85fbd04d6.png)

Reply:
Well, sure, because you're using a string and not the protectedString ID type. You should change the type of `prevPartId` to be `PartInstanceId | null` (since either it's going to be an ID or will be null) and then you can safely do the comparison without any unprotecting.